core/tasks.py

```diff
@@ -220,6 +219,10 @@ def deferred(self, *args, **kwargs):
                                             retryCount))
         cmd, data = json.loads(req.body, object_hook=jsonDecodeObjectHook)
         funcPath, args, kwargs, env = data
+        if conf.get("viur.tasks.logs", False):
+            logging.info("Call Task %s with CMD = %s" % (funcPath,cmd))
+            logging.info("Call Task Arguments args = %s kwargs = %s" % (str(args), str(kwargs)))
+            logging.info("Call Task Env = %s" % (str(env)))
```

Comment:
Can you please reduce to amount of logging.info calls to just one with all required information?
```suggestion
            logging.info(f"Call task {funcPath} with cmd={cmd!r} args={args!r} kwargs={kwargs!r} env={env!r}")
```